src/util.lisp

```diff
@@ -65,3 +68,70 @@
       (setf *compile-verbose* nil)
       (setf *compile-print* nil)
       (asdf:load-system sys :verbose nil))))
+
+(defun run-git-command (command &optional directory)
+  "Run the git COMMAND in the specified DIRECTORY."
+  (uiop:run-program (concatenate 'string "git " command)
+                    :directory (if directory (namestring directory))
+                    :wait t)
+  t)
+
+(defun run-hg-command (command)
+  "Run the Mercurial COMMAND."
+  (uiop:run-program (concatenate 'string "hg " command) :wait t :output :interactive)
+  t)
+
+(defun command-output (command directory)
```

Comment:
This is nasty, but I couldn't figure out how else to capture `stdout` from `uiop:run-program`.  It's easy with `sb-ext:run-program`, but UIOP's abstractions make it a lot less easy.